setup.py

```diff
@@ -40,12 +40,10 @@
         'requests',
         'PyJWT'
     ],
-    extras_require={
-        'test': [
-            'mock',
-            'requests_mock',
-            'future'
-        ],
-    },
+    tests_require=[
```

Comment:
I changed this because this is the only was I could get `python setup.py test` to run. Also, it seemed like a more appropriate place to put the test dependencies.

Reply:
that makes sense